docs/user-guide/lns-module-configuration.md

```diff
@@ -0,0 +1,38 @@
+# Network Server IoT Edge Module configuration
```

Comment:
Please have a look at the basics station configuration file header in order to remove the useless navigation sidebar on the right (useless as we don't have a TOC here and we could have more space for the table itself)